Tests/XcodeGenKitTests/ProjectGeneratorTests.swift

```diff
@@ -571,6 +571,89 @@ func projectGeneratorTests() {
                 try project.expectFile(paths: ["Sources/A"], names: ["A"], buildPhase: .resources)
                 try project.expectFileMissing(paths: ["Sources", "A", "a.swift"])
             }
+
+            $0.it("adds files to correct build phase") {
+                let directories = """
+                  A:
+                    - file.swift
+                    - file.xcassets
+                    - file.h
+                    - Info.plist
+                    - file.xcconfig
+                  B:
+                    - file.swift
+                    - file.xcassets
+                    - file.h
+                    - Info.plist
+                    - file.xcconfig
+                  C:
+                    - file.swift
+                    - file.m
+                    - file.mm
+                    - file.cpp
+                    - file.c
+                    - file.S
+                    - file.h
+                    - file.hh
+                    - file.hpp
+                    - file.ipp
+                    - file.tpp
+                    - file.hxx
+                    - file.def
+                    - file.xcconfig
+                    - file.entitlements
+                    - file.gpx
+                    - file.apns
+                    - file.123
+                    - file.xcassets
+                    - Info.plist
```

Comment:
What happens if it's `MyApp_Info.plist` instead of `Info.plist` ?

Reply:
It’s treated as a resource. Would be a good test case as well 👍

Reply:
I was about to comment the same. I wouldn't assume that `Info.plist` is the name of the target info.plist file. Is it feasible to get a list of all the info.plists defined in the yaml and us that to filter these .plist out from the resources?

Reply:
Yeah it's a tricky problem. I think it's fair to say that if we find a `Info.plist`, it's being used for that purpose. But yes, some projects may also have other names. It would be possible to check all the build settings for any `INFOPLIST_FILE` settings, and then ignore those. I've created a seperate issue for this #207